CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 This is a changelog for Piwik platform developers. All changes for our HTTP API's, Plugins, Themes, etc will be listed here.
 
+## Piwik 2.13.0
+
+### Deprecations
+* The method `Piwik\Archive::getBlob()` has been deprecated and will be removed from June 1st 2015. Use one of the methods `getDataTable*()` methods instead.
+
```

Comment:
That method is not used and not needed. It doesn't really make much sense to use it for now. Let's deprecate it, remove it for Piwik 3.0 or so and introduce something that is more helpful when needed.